english/Booyah/booyah.md

```diff
@@ -2,3 +2,5 @@ I finally understand git and gitHub. I actually understand branches and merges.
 
 What am I going to do with my new found knowledge?  
 The world is now my [oyster.](https://en.wikipedia.org/wiki/Pacific_oyster)
+
+You now have [diarrhea.](http://funnyasduck.net/wp-content/uploads/2012/11/funny-fry-futurama-fart-diarrhea-meme-pics.jpg)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.